components/typography.css

```diff
@@ -1,11 +1,26 @@
+@import "./base.css";
+@import "./utility.css";
 
- @import "./base.css";
- @import "./utility.css";
+.typography-container {
+  margin: 2rem 0rem;
+  justify-items: center;
+  background-color: var(--component-secondary-color);
+  padding: 2rem;
+  border-radius: 10px;
+}
 
- .typography-container{
-    margin: 2rem 0rem;
-    justify-items: center;
-    background-color: var(--component-secondary-color);
-    padding: 2rem;
-    border-radius: 10px;
- }
\ No newline at end of file
+.small-text {
```

Comment:
please try to use utility class for these styles. this was suggested in most of the Review comments in first submission.

Reply:
I didn't get you, this isn't a utility class for text?

Reply:
Its not utlity class its specific to typography for small text